### PR TITLE
[ i18n sprint ] move qr codes page translations to n3 configurations

### PR DESCRIPTION
**Pull request by litvinovg**
[VIVO companion PR](https://github.com/vivo-project/VIVO/pull/3777)

# What does this pull request do?
This pull request is one of many efforts to improve language neutral Freemarker templates and thus eliminate the need to support language specific templates
Created FixedHTMLDataGetters for "About qr codes" page
Removed language specific Freemarker templates for "About qr codes" page

### CODE REVIEW

File: de_DE/home/src/main/resources/rdf/i18n/de_DE/display/firsttime/aboutQrCodes.n3 [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    a   <java:edu.cornell.mannlib.vitro.webapp.utils.dataGetter.FixedHTMLDataGetter> ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#htmlValue>
+        """
+        <h2>Über QR-Codes</h2>
+
+		<h3>Was ist QR?</h3>
+
+		<p>QR, was für <i>Quick Response</i> steht, ist ein Standard für zweidimensionale Barcodes, die von mobilen Geräten mit Kameras (z.B. Smartphones) gelesen werden können.</p>
+
+		<h3>Welche Art von Daten sind in einem QR-Code enthalten?</h3>
```
Comment (bkampe) [inline de_DE/home/src/main/resources/rdf/i18n/de_DE/display/firsttime/aboutQrCodes.n3]:
```suggestion
		<h3>Welche Art von Daten ist in einem QR-Code enthalten?</h3>
```

File: de_DE/home/src/main/resources/rdf/i18n/de_DE/display/firsttime/aboutQrCodes.n3 [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+		<p>QR-Codes können viele verschiedene Arten von Daten enthalten. Der QR-Code auf der Profilseite einer Person in VIVO kodiert Informationen über diese Person im Format <i>vCard</i>.  Dies ist eine Art digitale Visitenkarte, die Informationen über den Namen, die Adresse, die Telefonnummer, die E-Mail-Adresse und mehr enthält.</p>
+
+		<p>Wenn Sie in VIVO auf den QR-Code einer Person klicken, gelangen Sie zur Seite <i>QR-Export</i> für diese Person. Diese Seite enthält den vCard QR-Code auf der Startseite sowie einen QR-Code, der einen einfachen Hyperlink zum VIVO-Profil des Benutzers enthält. Ebenfalls enthalen ist ein HTML-Tag für jeden Code, der in eine Webseite eingebettet werden kann, um diesen QR-Code in andere Seiten einzubinden.</p>
+
+		<h3>Wie lese ich einen QR-Code?</h3>
+
+		<p>Es gibt Anwendungen, die Sie für das iPhones und Android-Handys herunterladen können. Mit diesen können Sie die Kamera Ihres Handys verwenden, um mit QR-Codes zu interagieren, die auf Objekten in der realen Welt platziert sind.</p>
```
Comment (bkampe) [inline de_DE/home/src/main/resources/rdf/i18n/de_DE/display/firsttime/aboutQrCodes.n3]:
```suggestion
		<p>Es gibt Anwendungen, die Sie für iPhones und Android-Handys herunterladen können. Mit diesen können Sie die Kamera Ihres Handys verwenden, um mit QR-Codes zu interagieren, die auf Objekten in der realen Welt platziert sind.</p>
```

**Review by bkampe — CHANGES_REQUESTED** [review]
Minor spelling optimization

File: es/home/src/main/resources/rdf/i18n/es/display/firsttime/aboutQrCodes.n3 [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        """
+        <h2>Acerca de los c&oacute;digos QR</h2>
+
+		<h3>Qu&eacute; es QR?</h3>
+
+		<p>QR, siglas de <i>Quick Response</i>, es un est&aacute;ndar para los c&oacute;digos de barras de dos dimensiones que pueden ser le&iacute;dos por los dispositivos m&oacute;viles equipados con c&aacute;maras (como un tel&eacute;fono inteligente).</p>
+
+		<h3>Qu&eacute; tipo de dato es un c&oacute;digo QR?</h3>
+
+		<p>Los c&oacute;digos QR pueden codificar diferentes tipos de datos. El c&oacute;digo QR de la p&aacute;gina del perfil de una persona en VIVO codifica la informaci&oacute;n acerca de esa persona en formato <i>vCard</>. Se trata de una especie de tarjeta de presentaci&oacute;n digital, que contiene informaci&oacute;n sobre el nombre de la persona, direcci&oacute;n, n&uacute;mero de tel&eacute;fono, direcci&oacute;n de correo electr&oacute;nico y mucho m&aacute;s.</p>
```
Comment (chenejac) [inline es/home/src/main/resources/rdf/i18n/es/display/firsttime/aboutQrCodes.n3]: closing i tag is missing </i>, therefore the rest of the text is presented as italic in web browser

File: fr_CA/home/src/main/resources/rdf/i18n/fr_CA/display/firsttime/aboutQrCodes.n3 [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#saveToVar>
+                "aboutQrCodesPage" .
+about:about_qr_codes
+    a   <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#Page> ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#hasDataGetter>
+        about:about_qr_codes_dg ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#title>
+        "About QR Codes"@fr-CA ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#urlMapping>
+        "/about_qrcode" .
```
Comment (chenejac) [inline fr_CA/home/src/main/resources/rdf/i18n/fr_CA/display/firsttime/aboutQrCodes.n3:38]: not translated in French. There is translation in the deleted ftl file which should be copied here.

**Review by chenejac — CHANGES_REQUESTED** [review]

File: es/home/src/main/resources/rdf/i18n/es/display/firsttime/aboutQrCodes.n3 (hunk 3 above)
Comment (litvinovg) [reply es/home/src/main/resources/rdf/i18n/es/display/firsttime/aboutQrCodes.n3]: fixed

File: fr_CA/home/src/main/resources/rdf/i18n/fr_CA/display/firsttime/aboutQrCodes.n3 (hunk 4 above)
Comment (litvinovg) [reply fr_CA/home/src/main/resources/rdf/i18n/fr_CA/display/firsttime/aboutQrCodes.n3:38]: The text is from French template https://github.com/vivo-project/VIVO-languages/pull/113/files#diff-6eecbffaa25d76c4bf33300d2c177e871cfc0148fdb1f956e1a4961a157bdc88

**Review by chenejac — APPROVED** [review]

**Review by bkampe — APPROVED** [review]
Works like expected

File: ru_RU/home/src/main/resources/rdf/i18n/ru_RU/display/firsttime/aboutQrCodes.n3 [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        """
+        <h2>Про QR-коды</h2>
+
+		<h3>Что такое QR-код?</h3>
+
+		<p>QR (от <i>Quick Response [быстрый ответ]</i>) это стандарт двумерного графического кода, который может считываться имеющими камеру мобильными устройствами такими как смартфоны.</p>
+
+		<h3>Какого рода информация может содержаться в QR-коде?</h3>
+
+		<p>QR-код может хранить данные различного типа. QR-код на личной странице в VIVO содержит персональную информацию в формате <i>vCard</i>. Это своего рода цифровая визитная карточка с именем, адресом, телефоном, аэлектронной почтой и иными данными определёного лица.</p>
```
Comment (mmannings) [inline ru_RU/home/src/main/resources/rdf/i18n/ru_RU/display/firsttime/aboutQrCodes.n3]:
```suggestion
		<p>QR-код может хранить данные различного типа. QR-код на личной странице в VIVO содержит персональную информацию в формате <i>vCard</i>. Это своего рода цифровая визитная карточка с именем, адресом, телефоном, электронной почтой и иными данными определёного лица.</p>
```

File: ru_RU/home/src/main/resources/rdf/i18n/ru_RU/display/firsttime/aboutQrCodes.n3 [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+		<p>Щёлкнув по чьему-либо QR-коду в VIVO, вы попадаете на страницу <i>Экспортировать QR-код</i>. На этой странице будет тот же QR-код цифровй визитки в формате vCard, что был на предыдущей странице, а кроме того QR-код с гиперссылкой страницу с „профилем“ данного пользователя в системе VIVO. Также там будет HTML-код, пригодный для встраивания этих же QR-кодов в другие сайты.</p>
```
Comment (mmannings) [inline ru_RU/home/src/main/resources/rdf/i18n/ru_RU/display/firsttime/aboutQrCodes.n3]:
```suggestion
		<p>Щёлкнув по чьему-либо QR-коду в VIVO, вы попадаете на страницу <i>Экспортировать QR-код</i>. На этой странице будет тот же QR-код цифровой визитки в формате vCard, что был на предыдущей странице, а кроме того QR-код с гиперссылкой страницу с „профилем“ данного пользователя в системе VIVO. Также там будет HTML-код, пригодный для встраивания этих же QR-кодов в другие сайты.</p>
```

**Review by mmannings — APPROVED** [review]
Works as intended.

**Pull request merged**